ui/src/views/Provisioners/ViewWorkers/index.jsx

```diff
@@ -59,6 +60,19 @@ export default class ViewWorkers extends Component {
     selectedAction: null,
   };
 
+  componentDidMount() {
+    const query = parse(this.props.location);
+
+    if (query.search === '?filterBy=quarantined') {
+      const target = {
+        value: 'Quarantined',
+        name: '',
+      };
+
+      this.handleFilterChange({ target });
+    }
+  }
```

Comment:
We should be able to avoid having a `componentDidMount` by adding a quarantined parameter to the inital graphql query

https://github.com/taskcluster/taskcluster/blob/c57687672e50d827cc5b62acc9884841fb916535/ui/src/views/Provisioners/ViewWorkers/index.jsx#L25-L36

Reply:
Okay.